network-api/networkapi/buyersguide/models.py

```diff
@@ -191,6 +191,10 @@ class Product(models.Model):
         max_length=2,
     )
 
+    privacy_policy_reading_level_url = models.URLField(
+        blank="True"
```

Comment:
Can we add a `max_length` and `help_text`?

Reply:
`max_length` for URL?